README.md

```diff
@@ -9,16 +9,33 @@ A mongo mocking library with an ephemeral MongoDB running in memory.
 pip install pymongo-inmemory
 ```
 
+## Configuration
+| Config param     | Description                                                          | Optional? | Default                                          |
+|------------------|----------------------------------------------------------------------|-----------|--------------------------------------------------|
+| mongo_version    | Which MongoD version to download and use.                            | Yes       | 4.0.10                                           |
+| mongod_port      | Override port preference.                                            | Yes       | Automatically determined between 27017 and 28000 |
+| operating_system | This makes sense for Linux setting, where there are several flavours | Yes       | Automatically determined (Generic for Linux)     |
+
+
 ## Usage
-Insert a new section to your project's `setup.cfg` for the operating system and mongo
+### Configure
+There are several ways you can configure `pymongo_inmemory`.
+
+1. Insert a new section to your project's `setup.cfg` for the operating system and mongo
 version you want to spin up:
-```ini
-[pymongo_inmemory]
-mongo_version = 4.0
-operating_system = osx
-```
+    ```ini
+    [pymongo_inmemory]
+    mongo_version = 4.0
```

Comment:
should it be `4.0.10` ?

Reply:
This is a good question. I was intending it to work like SemVer but I haven't tested it indeed does. Even so, just leaving out the patch version shouldn't mean that latest patch will be installed. I will tackle this in another ticket.

Currently though, 4.0 will install latest 4.0